### PR TITLE
For #26414 - Have the Pocket spocs feature as part of the homescreen sections experiments

### LINKED ISSUE #26414: Have the Pocket sponsored feature be part of the homescreen experiments
Currently the Pocket sponsored feature is a Nimbus feature on it's own - https://github.com/mozilla-mobile/fenix/blob/main/.experimenter.yaml#L61.

It would make more sense to have the feature be part of the other homescreen sections already controllable through Nimbus - https://github.com/mozilla-mobile/fenix/blob/f356df785c6b179953ddb1a35907f1224c4377d3/.experimenter.yaml#L18-L20

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21258)

### PR DESCRIPTION
**Pull request by Mugurell**
Just a small refactoring of a feature not yet launched or part of Nimbus.

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #26414

### CODE REVIEW

File: nimbus.fml.yaml [hunk 1]
```diff
@@ -33,6 +34,12 @@ features:
             "contile-top-sites": false,
           }
         }
+      - channel: developer
+        value: {
+          "sections-enabled": {
+            "pocket-sponsored-stories": true,
+          }
+        }
```
Comment (Mugurell) [inline nimbus.fml.yaml:42]: From https://experimenter.info/fml-spec/ I understand this patches the default configuration and so there's no need to provide values for other sections for which I'm not interested to change their behavior.

Comment (Amejia481) [reply nimbus.fml.yaml:42]: 👍🏽

**Review by Amejia481 — APPROVED** [review]
This looks good!
Let's just rebase :)

**Comment by Mugurell** [comment]
`test-debug` failing because of https://github.com/mozilla-mobile/fenix/issues/26380.
Restarting CI.

**Comment by Mugurell** [comment]
`build-android-test-debug` failed because of 
> [task 2022-08-16T13:14:01.399Z] > Task :app:getDependencies
[task 2022-08-16T13:14:01.399Z] Failed to resolve OSS licenses for betaCompileClasspath.
[task 2022-08-16T13:14:01.409Z] org.gradle.api.artifacts.ResolveException: Could not resolve all dependencies for configuration ':app:betaRuntimeClasspath'.

Restarting task.

**Pull request merged**